Assets/Editor/CreatureLinker.cs

```diff
@@ -91,6 +91,21 @@ public static void LinkCreatureScripts()
                 if (healthScript == null)
                 {
                     healthScript = unitPrefab.AddComponent<HealthScript>();
+                    
+                    var healthAnimatorField = healthScript.GetType().GetField("enemyAnimator");
+                    if (healthAnimatorField != null)
+                    {
+                        healthAnimatorField.SetValue(healthScript, unitPrefab.GetComponent<Animator>());
+                    }
+
+                    var healthBarField = healthScript.GetType().GetField("healthBar");
+                    if (healthBarField != null)
+                    {
+                        // TODO: Make this not shitty and fragile
+                        string healthBarPath = "Assets/AlienScifiCharacter/InnerHealthBar.prefab";
```

Comment:
should this be a lookup that errors out instead for here and below?

Reply:
Good point made a ticket

Reply:
https://trello.com/c/ifwLITcs/96-make-linking-less-fragile